tests/utils_tests/test_encoding.py

```diff
@@ -49,20 +50,30 @@ def test_force_bytes_strings_only(self):
         today = datetime.date.today()
         self.assertEqual(force_bytes(today, strings_only=True), today)
 
-    def test_smart_text(self):
+    def test_force_bytes_encoding(self):
+        error_msg = 'This is an exception, voilà'.encode('utf-8')
+        result = force_bytes(error_msg, encoding='ascii', errors='ignore')
+        self.assertEqual(result, b'This is an exception, voil')
+
+    def test_smart_bytes(self):
         class Test:
             def __str__(self):
                 return 'ŠĐĆŽćžšđ'
 
-        class TestU:
```

Comment:
I'm not positive whether or not `TestU` is obsolete but there were more differences between `Test` and `TestU` when Python 2 was supported.